src/main/resources/assets/blocky-bubbles/lang/ru_ru.json

```
@@ -0,0 +1,13 @@
+{
+  "blocky-bubbles.options.culling_awareness.enum.bubble_column": "Пузырьковая колонка (подобная Java)",
+  "blocky-bubbles.options.culling_awareness.enum.non_air": "Невоздушные (паритет)",
+  "blocky-bubbles.options.culling_awareness.enum.none": "Без отбраковки",
+  "blocky-bubbles.options.culling_awareness.name": "Режим отбраковки",
+  "blocky-bubbles.options.culling_awareness.tooltip": "Какие блоки должны вызывать отбраковку соседних граней колонны пузырей.",
+  "blocky-bubbles.options.enable_animations.name": "Включить анимацию",
+  "blocky-bubbles.options.enable_animations.tooltip": "Следует ли анимировать текстуры колонны пузырей (требуется перезагрузка ресурсов).",
+  "blocky-bubbles.options.quality.name": "Колонны пузырей",
+  "blocky-bubbles.options.quality.tooltip": "Контролирует качество колонны пузырей.",
```

В оригинальной строке используется множественное число (bubble columns), + следует согласовывать перевод `Controls` с уже существующими строками официального перевода, такими как [#5342178](https://crowdin.com/editor/minecraft/10038/enus-ru#5342178), [#5343563](https://crowdin.com/editor/minecraft/10038/enus-ru#5343563), [#5344423](https://crowdin.com/editor/minecraft/10038/enus-ru#5344423), [#5344427](https://crowdin.com/editor/minecraft/10038/enus-ru#5344427) и т.п.
```suggestion
  "blocky-bubbles.options.quality.tooltip": "Влияет на качество колонн пузырей.",
```